distribution/Directory.Build.props

```diff
@@ -1,10 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <PropertyGroup Label="Metadata configuration">
+    <OrganizationName>atc-net</OrganizationName>
+    <RepositoryName>atc-cosmos</RepositoryName>
```

Comment:
Don't think this should be default values - suggestion:

- <OrganizationName>[INSERT-YOUR-VALUE]</OrganizationName>
- <RepositoryName>[INSERT-YOUR-VALUE]</RepositoryName>